src/installer/installer.go

```diff
@@ -246,14 +247,13 @@ func (i *installer) waitForBootkube(ctx context.Context) {
 			return
 		case <-time.After(time.Second * time.Duration(5)):
 			// check if bootkube is done every 5 seconds
-			out, _ := i.ops.ExecPrivilegeCommand(false, "bash", "-c", "systemctl status bootkube.service | grep 'bootkube.service: Succeeded' | wc -l")
-			if out == "1" {
+			if _, err := os.Stat("/opt/openshift/.bootkube.done"); err == nil {
```

Comment:
Unsure this will work since this get executed in the container and bootkube is running on the host and writing the file there, need to make sure this directory is mounted.
However you can do this:
 ```suggestion
			out, _ := i.ops.ExecPrivilegeCommand(false, "stat", "/opt.openshift/.bootkube.done")
```
Also easier to UT